src/transaction/log_recovery_analysis.cpp

```diff
@@ -1628,18 +1657,21 @@ static void
 log_rv_analysis_record_on_tran_server (THREAD_ENTRY *thread_p, LOG_RECTYPE log_type, int tran_id, LOG_LSA *log_lsa,
 				       LOG_PAGE *log_page_p, LOG_LSA *prev_lsa, log_recovery_context &context)
 {
+  bool is_mvcc = false;
   switch (log_type)
     {
-    case LOG_UNDOREDO_DATA:
-    case LOG_DIFF_UNDOREDO_DATA:
-    case LOG_UNDO_DATA:
-    case LOG_REDO_DATA:
     case LOG_MVCC_UNDOREDO_DATA:
     case LOG_MVCC_DIFF_UNDOREDO_DATA:
     case LOG_MVCC_UNDO_DATA:
     case LOG_MVCC_REDO_DATA:
+      is_mvcc = true;
```

Comment:
I think it is easier to maintain with break and explicit call to `log_rv_analysis_undo_redo` with false/true as appropriate. In the end, no line of code is saved with fall-through.

Reply:
You're right, fixed.